### ISSUE TITLE
Reduce reliance on new processes for LoRa communication

### ISSUE THREAD

**Issue by j-rewerts** [CityofEdmonton/pedestrian-counter#21]
Currently, we create a new Process every time we want to send a message over LoRa. There should be a couple alternatives to doing it this way.

1. We create a message process on start of our Python script. When we want to send a message, we pipe a string to the LoRa process.
2. We write Python bindings for the LMIC library. Our C++ wrapper around LMIC doesn't actually do that much, so this would be a very elegant solution. I bet others in the LoRa community would appreciate if we did this.

**Comment by kylecjo** [comment]
I spent some time testing sending "Hello, world" straight from thethingsnetwork-send-v1 file, and I discovered something. Each process we start actually tries to send a LoRa transmission every 15 seconds. 

Line 145 thethingsnetwork-send-v1.cpp
`    // Schedule a timed job to run at the given timestamp (absolute system time)
    os_setTimedCallback(j, os_getTime() + sec2osticks(15), do_send);`

A successfully sent message will run the following line:

Line 100 thethingsnetwork-send.v1.cpp
`        fprintf(stdout, "Event EV_TXCOMPLETE, time: %d\n", millis() / 1000);`

However, not all of these successfully sent messages are received by the gateway.  In fact, most of the scheduled jobs to send a message will actually fail and print this line

Line 136 thethingsnetwork-send-v1.cpp
`fprintf(stdout, "OP_TXRXPEND, not sending");`

And these processes will actually completely fail and hang due to failing one of the many ASSERT statements in radio.c. These failures occur anywhere between immediately and a few minutes. If we had just one process, we could send at most a few minutes of data before it stopped sending. 

So somehow, we've been able to send data continuously purely because we are starting multiple processes. It doesn't matter if one process completely fails because it just gets  closed and the next one hopefully sends a successful message before it gets closed.  This is probably why increasing the frequency of processes also increases the likelihood of a successful message. More processes means higher chance the process doesn't fail.
